app/views/global_user_roles/edit.html.haml

```diff
@@ -0,0 +1,27 @@
+= content_for :h1 do
```

Comment:
I got the `map_users` partial closer to `global_user_roles/edit` here 9cf4b08534ec6212e0241065a2db02ab6ea2f223 but did not try to extract common parts. If we go with allowing multiple facility roles for a user, nearly the entire form would be the same. Want to leave this as a `TODO`?